web-console/src/views/workbench-view/max-tasks-button/max-tasks-button.tsx

```diff
@@ -132,21 +119,36 @@ export const MaxTasksButton = function MaxTasksButton(props: MaxTasksButtonProps
               label={capitalizeFirst(taskAssigment)}
               submenuProps={{ style: { width: 300 } }}
             >
-              {TASK_ASSIGNMENT_OPTIONS.map(t => (
-                <MenuItem
-                  key={String(t)}
-                  icon={tickIcon(t === taskAssigment)}
-                  text={
-                    <>
-                      <strong>{capitalizeFirst(t)}</strong>: {TASK_ASSIGNMENT_DESCRIPTION[t]}
-                    </>
-                  }
-                  labelElement={TASK_ASSIGNMENT_LABEL_ELEMENT[t]}
-                  shouldDismissPopover={false}
-                  multiline
-                  onClick={() => changeQueryContext({ ...queryContext, taskAssignment: t })}
-                />
-              ))}
+              <MenuItem
+                icon={tickIcon(taskAssigment === 'max')}
+                text={
+                  <>
+                    <strong>Max</strong>: uses the maximum possible tasks up to the specified limit.
+                  </>
+                }
+                shouldDismissPopover
+                multiline
+                onClick={() => changeQueryContext({ ...queryContext, taskAssignment: 'max' })}
+              />
+
+              <MenuItem
+                icon={tickIcon(taskAssigment === 'auto')}
+                text={
+                  <>
+                    <strong>Auto</strong>: Use the minimum number of tasks while{' '}
+                    <a
+                      href={TASK_DOCUMENTATION_LINK}
+                      target="_blank"
+                      onClick={e => e.stopPropagation()}
+                    >
+                      staying within constrain.
```

Comment:
Spelling: "constraints"

Reply:
Can't use an `<a>` element here because the <MenuItem> is itself an `a` and if you look at the JS console there is a nasty browser warning